docroot/profiles/favrskov/modules/features/other/feeds_job_importer/feeds_job_importer.feeds_tamper_default.inc

```diff
@@ -28,7 +29,7 @@ function feeds_job_importer_feeds_tamper_default() {
   $export['job_importer-description-find_replace_regex'] = $feeds_tamper;
 
   $feeds_tamper = new stdClass();
-  $feeds_tamper->disabled = FALSE; /* Edit this to true to make a default feeds_tamper disabled initially */
+  $feeds_tamper->disabled = TRUE; /* Edit this to true to make a default feeds_tamper disabled initially */
```

Comment:
@mnakov 

Same here - we have an alter of the date format, maybe now in the new endpoint we have different date formatting?

Reply:
Yes, different time formatting it is. Old format: "2019-03-31", New format: 31. mar 2019. This tamper is no longer working with the new date format, but I modified the view to render the final date format as it was in the old job listing. And we have the client approval for it. The client has reviewed all the visual changes on the feature demo.